src/util/RedisClient.js

```diff
@@ -0,0 +1,24 @@
+const { noop } = require('lodash');
+const { redisUrl } = require('./environmentVariables');
+const { promisify } = require('util');
+const { RedisClient: BaseRedisClient } = require('redis');
+const unifyOptions = require('redis/lib/createClient');
+
+module.exports = exports = class RedisClient extends BaseRedisClient {
```

Comment:
Nope. It's this entire structure.

Reply:
Specifically, the constructor. This is because the original RedisClient is not a class, but a funciton. So it kind of destroys everything the function does. I'm going to close this out for now and merge in a small amount of changes. It also feels like node-redis is a dead project for now?